server/request_handler.go

```diff
@@ -134,11 +134,11 @@ func (r *RpcRequestHandler) process() {
 	r.logger = r.logger.New("rpc_method", jsonReq.Method)
 
 	// Process single request
-	r.processRequest(client, jsonReq, origin, referer, isWhitehatBundleCollection, whitehatBundleId, urlParams, r.req.URL.String())
+	r.processRequest(client, jsonReq, origin, referer, isWhitehatBundleCollection, whitehatBundleId, urlParams, r.req.URL.String(), body)
```

Comment:
at some point we need to start passing in a struct :)

Reply:
Agreed. :). Luckily the refactor you suggested, which is implemented in https://github.com/flashbots/rpc-endpoint/pull/151/commits/d90fb9b1c8de38358dbf4e5c748e52b234462300 removes the additional parameter.